src/app.js

```diff
@@ -186,3 +186,21 @@ router.use("^/$", (req, res) => {
 });
 
 router.use(commonExpress.lib.errorHandling.redirectAsErrorToCallback);
+
+/* Server configuration */
+const server = app.listen(PORT, "0.0.0.0");
+
+// AWS recommends the keep-alive duration of the target is longer than the idle timeout value of the load balancer (default 60s)
+// to prevent possible 502 errors where the target connection has already been closed
+// https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-troubleshooting.html#http-502-issues
+server.keepAliveTimeout = 65000;
```

Comment:
What do we want the general timeout to be? 

https://nodejs.org/docs/latest-v20.x/api/http.html#servertimeout

By default it is `0` - no timeout

Reply:
server.keepAliveTimeout[#](https://nodejs.org/docs/latest-v20.x/api/http.html#serverkeepalivetimeout)
Added in: v8.0.0
[<number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) Timeout in milliseconds. Default: 5000 (5 seconds).

Reply:
> server.timeout#
> <number> Timeout in milliseconds. Default: 0 (no timeout)
> The number of milliseconds of inactivity before a socket is presumed to have timed out.
>
> A value of 0 will disable the timeout behavior on incoming connections.
> 
> The socket timeout logic is set up on connection, so changing this value only affects new connections to the server, not any existing connections.

We don't have to change it now - and we should aim to only change 1 thing at a time anyway - but we should be aware that there's more than 1 timeout in play here

Reply:
Aye agreed.